modules/consensus.go

```diff
@@ -172,11 +172,11 @@ type (
 		// current block is presented to the subscriber.
 		ConsensusSetDigestSubscribe(ConsensusSetDigestSubscriber)
 
-		// EarliestChildTimestamp returns the earliest timestamp that is acceptable
-		// on the current longest fork according to the consensus set. This is a
-		// required piece of information for the miner, who could otherwise be at
+		// MinimumValidChildTimestamp returns the earliest timestamp that is
+		// valid on the current longest fork according to the consensus set. This is
+		// a required piece of information for the miner, who could otherwise be at
 		// risk of mining invalid blocks.
-		EarliestChildTimestamp(types.BlockID) (types.Timestamp, bool)
+		MinimumValidChildTimestamp(types.BlockID) (types.Timestamp, bool)
```

Comment:
> Do you feel strongly about this name change?

EarliestChildTimestamp is a misleading name to me because I think a reasonable reader would interpret that to be "of the blocks existing children, give me the timestamp of the earliest child." I think that MinimumValidChildTimestamp potentially also has this ambiguity, but I feel like it more clearly indicates the function's purpose.

I'm open to other names, but I feel like `MinimumValidChildTimestamp` is an improvement. I get that it's longer, but it's only 4 characters longer.